pkgs/mime/doc/media_types.md

```diff
@@ -592,20 +586,19 @@ Supported media types and file extensions.
 | `application/xproc+xml` | `xpl` | |
 | `application/xslt+xml` | `xslt` | |
 | `application/xspf+xml` | `xspf` | |
-| `application/xv+xml` | `xvml` | `mxml`, `xhvml`, `xvm` |
+| `application/xv+xml` | `mxml` | `xhvml`, `xvm`, `xvml` |
 | `application/yang` | `yang` | |
 | `application/yin+xml` | `yin` | |
 | `application/zip` | `zip` | |
-| `audio/aac` | `aac` | |
 | `audio/adpcm` | `adp` | |
-| `audio/basic` | `snd` | `au` |
+| `audio/basic` | `au` | `snd` |
 | `audio/midi` | `mid` | `kar`, `midi`, `rmi` |
-| `audio/mp4` | `m4a` | `m4b`, `mp4a` |
+| `audio/mp4` | `m4a` | `mp4a` |
 | `audio/mpeg` | `mpga` | `m2a`, `m3a`, `mp2`, `mp2a`, `mp3` |
```

Comment:
See #2048 
We might not want to trust that the first extension is the best default extension.
They seem to be ordered alphabetically, so the file is a mapping from extension to file type, *not* the other direction.

For some file types, some extensions are definitely better as default than others, like `mp3` or `ogg`.
For others, it doesn't even make sense to have a default. See fx `application/x-msdownload` - those are completely different kinds of files, and assigning the wrong extension would be a mistake, but to a *web server*, they are all things you want to download on Windows.